### DISCUSSION TITLE
https://plugin-auto-catalog.vuejs.press/zh/config.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#543] [Announcements]
# https://plugin-auto-catalog.vuejs.press/zh/config.html

插件选项 level 类型: 1 2 3; 默认值: 3; 目录项级别的最大深度。 仅在你使用内置目录组件时可用。 index 类型: boolean; 默认值: false; 目录是否显示索引 exclude 类型: (RegExp | string)[]; 默认值: []; 不会自动生成的页面路径。 frontmatter 类型: (path: s...

https://plugin-auto-catalog.vuejs.press/zh/config.html

**Comment by ChaXxl** [comment]
可以设置让目录在页面左侧显示吗，真的很需要，谢谢啦！

**Reply by ChaXxl** [reply]
以及，能不能让目录上的 "此页内容" 改为显示 markdown 文件名称呢，或者可以自定义

**Reply by Mister-Hope** [reply]
你说的这个东西是 hope 主题提供的 toc，不是插件提供的目录组件。
